testing/trino-testing-services/src/main/java/io/trino/DockerImagesDefaults.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino;
+
+import static java.lang.String.format;
+
+public final class DockerImagesDefaults
```

Comment:
By itself, this seems like a spurious change that just introduces an artificial dependency between the trino-testing-services and product-tests-launcher modules. It should go in the commit that uses these fields.

However, I'm not fully convinced about the value of adding this "centralization". It just couples code across modules unnecessarily.

Reply:
I requested this change with the idea that we'd want to ensure that both PTs and integration tests (the ones being added here) are always ran on the same image. Maybe we can ensure this some other way too?

Reply:
I will revert it back for now until we will find appropriate approach for that. Seems like a nice candidate for other small PR.

Reply:
Reverted